docs/api-reference.rst

```diff
@@ -0,0 +1,18 @@
+.. _api-reference:
+
+API
+===
+
+Here you can find the API of some of the important classes and methods in microraiden.
+This documentation is autogenerated from the docstrings and type-hints
+in the source code. If you have further questions look in the source-code
+directly.
+
+
+Client
+------------------
+
+.. autoclass:: microraiden.client.client.Client
+    :members:
+
+
```

Comment:
Any reason why only the `Client` api is here? I saw you created an entire API folder for autogenerated docs from docstrings.